.github/workflows/release.yml

```diff
@@ -0,0 +1,42 @@
+name: release
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        default: "0.0.1"
```

Comment:
I don't think having a default value here makes sense

I also think it'd be nice to support the version being specified with or without a "v" prefix and it producing the same result either way. I release so infrequently here I'll forget which is correct